rllib/examples/parallel_evaluation_and_training.py

```diff
@@ -36,6 +37,20 @@
     default=100.0,
     help="Reward at which we stop training.")
 
+
+class AssertNumEvalEpisodesCallback(DefaultCallbacks):
+    def on_train_result(self, *, trainer, result, **kwargs):
+        # Make sure we always run exactly n evaluation episodes,
+        # no matter what the other settings are (such as
+        # `evaluation_num_workers` or `evaluation_parallel_to_training`).
+        if "evaluation" in result:
+            hist_stats = result["evaluation"]["hist_stats"]
+            assert len(hist_stats["episode_lengths"]) == \
```

Comment:
a random though, episode_lengths and num_episodes sound like 2 totally different thing.
assert that they should equal reads a bit confusing.
but they are probably part of our public api at this point? So understand if we can't do much about it.

Reply:
:+1: I clarified in the comment what we are doing here.